packages/router/src/utilities/utils.ts

```diff
@@ -19,6 +19,21 @@ export function arrayRemove<T>(arr: T[], func: (value: T, index?: number, obj?:
   return removed;
 }
 
+/**
+ * @internal
+ */
+export function arrayAddUnique<T>(arr: T[], values: T | T[]): T[] {
+  if (!Array.isArray(values)) {
+    values = [values];
+  }
+  for (const value of values) {
+    if (!arr.includes(value)) {
+      arr.push(value);
+    }
+  }
```

Comment:
We can have a shorter version for this:
```suggestion
  arr.push(...values.filter(v => !arr.includes(v));
```

Reply:
Haha, yes, indeed. Think it was late one night and I asked Copilot to create this func and then just didn't review it enough. Will fix.

Reply:
actually there's a difference in behavior between my suggestion and the original code where the original also does a unique check for incoming values.

Reply:
Yeah, I realized this as well out of the blue yesterday. Will be restored in the next push.